agent/src/beerocks/slave/beerocks_slave_main.cpp

```diff
@@ -534,8 +537,14 @@ int main(int argc, char *argv[])
             auto hostap_iface_elm = interfaces_map.find(slave_num);
             if ((hostap_iface_elm != interfaces_map.end()) &&
                 (g_son_slave_iface == hostap_iface_elm->second)) {
-                return run_son_slave(slave_num, beerocks_slave_conf, hostap_iface_elm->second, argc,
-                                     argv);
+                int no_vendor_specific = beerocks::bpl::cfg_get_no_vendor_specific();
```

Comment:
Clarifying what @tomereli commented earlier: normally, the only component making calls to bpl should be the platform_manager_thread. Although there are already a couple of calls to bpl from this file, they really should be exceptions.

So @tomereli's proposal is to add it to the ACTION_PLATFORM_SON_SLAVE_REGISTER_RESPONSE structure and fill it in in fill_platform_settings() in platform_manager_thread. The only thing to be checked is if it then arrives on time on the slave thread (i.e. before the no_vendor_specific config is used). I think it does, but needs to be checked.